docs/test_plan/proctored_exam.rst

```diff
@@ -0,0 +1,133 @@
+Proctored Exams
+===============
+
+Helpful ReadTheDocs Pages:
+
+- `Configuring a Proctoring Provider <https://edx.readthedocs.io/projects/edx-partner-course-staff/en/latest/proctored_exams/proctored_enabling.html#configuring-proctoring-provider>`_
+- `Creating a Proctored Exam <https://edx.readthedocs.io/projects/edx-partner-course-staff/en/latest/proctored_exams/pt_create.html#creating-a-proctored-exam>`_
```

Comment:
Neither of these links is used in this document.

Reply:
Ah these are copied over from [timed_exam.rst](https://github.com/edx/edx-exams/blob/main/docs/test_plan/timed_exam.rst), where it seems like they aren't used either. I see what you're saying though, I can have these links referenced in the setup instructions where it makes sense.

Reply:
I think this might be how they are referenced the other docs has them right next to the step to create a proctored exam. Maybe we just want to point back to this if these instructions are needed anywhere later in the doc. If you use headings you can add internal links within the doc like this https://github.com/openedx/edx-proctoring/blob/master/docs/testing/test_plan.md?plain=1#L220. That can also be good to avoid repeating yourself  if one test depends on the setup of another

Reply:
Ah, I see. So it's like reference materials? My bad - got confused!